test/date.test.js

```diff
@@ -152,17 +151,59 @@ describe('o-date', () => {
 		});
 
 		it('returns a result from ODate.timeAgo if ODate.isToday returns true', () => {
-			const oDateFormatReturn = "mocked timeAgo date";
+			const oDateTimeAgoReturn = "mocked timeAgo date";
 			spyOn(oDate, 'isNearFuture').and.returnValue(false);
 			spyOn(oDate, 'isFarFuture').and.returnValue(false);
 			spyOn(oDate, 'isToday').and.returnValue(true);
 
-			spyOn(oDate, 'timeAgo').and.returnValue(oDateFormatReturn);
+			spyOn(oDate, 'timeAgo').and.returnValue(oDateTimeAgoReturn);
 
-			expect(oDate.ftTime(someDate)).toBe(oDateFormatReturn);
+			expect(oDate.ftTime(someDate)).toBe(oDateTimeAgoReturn);
 			expect(oDate.timeAgo).toHaveBeenCalledWith(someDate, jasmine.any(Date), jasmine.any(Number));
 		});
 
+		it('returns a result from timeAgo if the publish date is less than 4 hours ago even if that date is also yesterday', ()=>{
+			const oDateTimeAgoReturn = "mocked timeAgo date";
+			spyOn(oDate, 'timeAgo').and.returnValue(oDateTimeAgoReturn);
+
+			let publishDatesInTheLast4Hours =
```

Comment:
Slightly inconsistent spacing but no big deal. Also I think nearly all of your variables can be `const` in here